styleguide.config.js

```diff
@@ -15,75 +15,94 @@ module.exports = {
 		module: {
 			rules: require('./webpack.dev.js').module.rules.filter(
 				rule => rule.use !== 'eslint-loader'
-			)
-		}
+			),
+		},
 	}),
 	exampleMode: 'expand',
 	usageMode: 'expand',
 	components: 'src/components/*/*.vue',
 	getComponentPathLine(componentPath) {
-		const name = path.basename(componentPath, '.vue');
-		return `import { ${name} } from '@nextcloud/vue/dist/Components/${name}';`;
+		const name = path.basename(componentPath, '.vue')
+		return `import { ${name} } from '@nextcloud/vue/dist/Components/${name}';`
```

Comment:
We should remove the semicolon here was well :)

Reply:
```suggestion
		return `import { ${name} } from '@nextcloud/vue/dist/Components/${name}'`
```